code/implementation/docker/machine/src/main/java/io/cattle/platform/docker/machine/process/MachinePreCreate.java

```diff
@@ -1,7 +1,9 @@
 package io.cattle.platform.docker.machine.process;
 
-import static io.cattle.platform.core.model.tables.PhysicalHostTable.*;
```

Comment:
Unnecessary import change!
I suggest changing your eclipse settings for "Number of static imports needed for .*..." to 1.
It's under Preferences > Java > Code Style > Organzine Imports.

I know that at least Darren and I have it set to 1.

I won't hold up the PR for this, so please just change it so that it doesn't happen next time.